packages/sources/src/addSourceWizard/hardcodedComponents/openshift/costMangement.js

```diff
@@ -0,0 +1,222 @@
+import { TextContent, Text, TextVariants, TextListItem, TextList, Button, ClipboardCopy, Popover, ButtonVariant } from '@patternfly/react-core';
+import { QuestionCircleIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const PrerequisiteDescription = () => (
+    <TextContent key='description'>
+        <Text component={TextVariants.p}>
+            Before you begin, install the following prerequisites if not present:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPText = () => (
+    <TextContent key='prerequisite-ocp'>
+        <Text component={TextVariants.p}>
+            On your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteSystemText = () => (
+    <TextContent key='prerequisite-system'>
+        <Text component={TextVariants.p}>
+            On a system with network access to your OpenShift cluster, install:
+        </Text>
+    </TextContent>
+);
+
+export const PrerequisiteOCPList = () => (
+    <TextContent>
+        <TextList component='ul' style={{ marginLeft: '16px', marginTop: '-16px' }}>
```

Comment:
Please, dont use inline styles, you can use PF4 class names https://www.patternfly.org/v4/documentation/core/utilities/spacing and it is strange to overrides default margins in a standard list. It should look according to default Patternfly design.